horizomer/utils/tests/test_tree.py

```diff
@@ -327,6 +328,34 @@ def test_build_taxdump_tree(self):
                              '(19,20)8)3)1;'])
         self.assertTrue(compare_topology(obs, exp))
 
+    def test_order_nodes(self):
+        """Test order nodes"""
+        tree1 = TreeNode.read(['(((a,b),(c,d,i)j),((e,g),h));'])
```

Comment:
I would add a more explicit comparison to a tree that is definitely sorted.

For instance, you should try to use this tree and see if this is ordered (it should return true)
```
expected_tree = TreeNode.read(['(((c,d,i)j, (a,b)),((e,g),h));'])
```

Reply:
@nujinuji Basically it means that you should provide several trees for each of which you know whether it is ordered for sure, and indicate that in comments.